### PR TITLE
Ourimpact DONE?

### PR DESCRIPTION
**Pull request by liannarice**
done?

### CODE REVIEW

File: css/impact.css [hunk 1]
```diff
@@ -205,7 +209,9 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 /* Desktop Only */
 @media screen and (min-width: 1024px) {
 	/* Add styles that apply specifically to desktop */
-  
+  .responsible-wrapper {
```
Comment (maxharrisnet) [inline css/impact.css]: I think this needs a little bit of padding on the left 

Screen Shot 2024-01-30 at 8 25 45 AM

File: impact.html [hunk 2]
```diff
@@ -251,7 +253,7 @@ <h2 class="h4-styles">
 
 			<!-- Responsible section -->
 			<section class="responsible-parent">
-				<div class="responsible-info wrapper">
+				<div class="responsible-info .responsible-wrapper">
```
Comment (maxharrisnet) [inline impact.html]: Remove the dot here

File: css/impact.css (hunk 1 above)
Comment (maxharrisnet) [reply css/impact.css]: This might be caused by the issue below. The dot should be removed from the class name in the HTML.

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
Almost there, just need to fix a typo

File: impact.html (hunk 2 above)
Comment (liannarice) [reply impact.html]: done

File: css/impact.css (hunk 1 above)
Comment (liannarice) [reply css/impact.css]: fixed

Comment (charlisable) [reply css/impact.css]: fixed the issue with typo and adjusted styles in css

**Review by maxharrisnet — APPROVED** [review]
Nice work 🎉

**Pull request merged**